src/build_workflow/build_args.py

```diff
@@ -44,15 +48,34 @@ def __init__(self) -> None:
             default=False,
             help="Build snapshot.",
         )
-        parser.add_argument("-c", "--component", type=str, help="Rebuild a single component.")
+        parser.add_argument(
+            "-c",
+            "--component",
+            dest="components",
+            nargs='+',
```

Comment:
This should be using `*` not `+`.

'+'. Just like '*', all command-line args present are gathered into a list. Additionally, an error message will be generated if there wasn’t at least one command-line argument present.

Since `--component` is not a required argument for out case.
This should be treat as 0 or more, not 1 or more.

Thanks.